frontends/precinct-scanner/src/screens/scan_error_screen.tsx

```diff
@@ -46,7 +48,7 @@ export function ScanErrorScreen({ error, isTestMode }: Props): JSX.Element {
     <ScreenMainCenterChild infoBar={false}>
       <TimesCircle />
       <CenteredLargeProse>
-        <h1>Scanning Error</h1>
+        <h1>Ballot Not Counted</h1>
         <p>{errorMessage}</p>
         <Text italic>Ask a poll worker for help.</Text>
```

Comment:
We should also probably prompt to remove ballot?

> Remove ballot and ask a poll worker for help.

Reply:
It's not guaranteed that the ballot will be held in the front in all of these cases. E.g. `plustek_error` could happen at any time.

Since this screen handles both invalid interpretations and other scanning errors, I'm not sure it even makes sense to always show this message. I suggest we move this message to be a part of the `switch` above so that we can determine when to show something about asking a poll worker or not.

Also, I'm realizing that other pages say "Ask a poll worker if you need help," which might be more appropriate in some cases when there's a mitigation that the voter can take (e.g. removing ballot and trying again). I think "Ask a poll worker for help" is better for cases when there's no voter mitigation available.

Happy to help distinguish which cases are which if that's helpful!

Reply:
All screens should ideally inform/empower the voter to resolve the issue themself before asking a poll worker for help.

If the voter removes their ballot, the error screen will no longer be displayed. So they will have to scan again before they can show a poll worker what the error is.

Jonah is adding the additional previews. We can determine where we should to explicitly prompt the voter to remove their ballot once these are available.